gammapy/data/tests/test_data_store.py

```diff
@@ -58,6 +58,15 @@ def test_DataStore_load(data_manager):
     events = data_store.load(obs_id=23037, filetype='events')
     assert isinstance(events, EventList)
 
+#@pytest.mark.xfail
+@requires_data('hess')
+def test_DataStore_load_all():
+    """Test loading data and IRF files via the DataStore"""
+    dir = str(gammapy_extra.dir) + '/datasets/hess-crab4'
+    data_store = DataStore.from_dir(dir)
+    event_lists = data_store.load_all(filetype='events')
+    assert event_lists[0]['ENERGY'][0] == 1.1156039
```

Comment:
I don't understand why the test failed here. I got this but this is the same value no?
```
gammapy/data/tests/test_data_store.py::test_DataStore_construction xfail
gammapy/data/tests/test_data_store.py::test_DataStore_filenames xfail
gammapy/data/tests/test_data_store.py::test_DataStore_load xfail
gammapy/data/tests/test_data_store.py::test_DataStore_load_all FAILED
gammapy/data/tests/test_data_store.py::test_DataStore_other xfail

=================================== FAILURES ===================================
___________________________ test_DataStore_load_all ____________________________

    @requires_data('hess')
    def test_DataStore_load_all():
        """Test loading data and IRF files via the DataStore"""
        dir = str(gammapy_extra.dir) + '/datasets/hess-crab4'
        data_store = DataStore.from_dir(dir)
        event_lists = data_store.load_all(filetype='events')
>       assert event_lists[0]['ENERGY'][0] == 1.1156039
E       assert 1.1156039 == 1.1156039

gammapy/data/tests/test_data_store.py:68: AssertionError
===================== 1 failed, 4 xfailed in 1.22 seconds ======================

``